utils/logging/format.go

```diff
@@ -133,7 +133,7 @@ func jsonLevelEncoder(l zapcore.Level, enc zapcore.PrimitiveArrayEncoder) {
 	enc.AppendString(Level(l).LowerString())
 }
 
-func consoleColorLevelEncoder(l zapcore.Level, enc zapcore.PrimitiveArrayEncoder) {
+func ConsoleColorLevelEncoder(l zapcore.Level, enc zapcore.PrimitiveArrayEncoder) {
```

Comment:
Should we also export `levelEncoder` and `jsonLevelEncoder` if we are exporting `consoleColorLevelEncoder`?